### PR TITLE
Analysis Tools - Implemented 2 test cases JPSURV Application

### PR DESCRIPTION
**Pull request by sgugulothu**
Analysis Tools - Implemented 2 test cases JPSURV Application

### CODE REVIEW

**Review by sonika-nih — APPROVED** [review]
Approved

File: src/test/java/ANALYSIS_TOOLS/JPSurv/stepsImplementations/JPSurvHomePagePlaywrightStepImp.java [hunk 1]
```diff
@@ -89,4 +89,16 @@ public static void selectBothNhLAndCHL() {
         PlaywrightUtils.page.locator("//label[@for='Non-Hodgkin Lymphoma']").click();
         PlaywrightUtils.page.locator("//label[@for='Chronic Myeloid Leukemia']").click();
     }
+
+    /**
+     * SELECTS THE RELAX PROPORTIONALITY AND CUTPOINT
+     *
+     * @param string
```
Comment (matakevin) [inline src/test/java/ANALYSIS_TOOLS/JPSurv/stepsImplementations/JPSurvHomePagePlaywrightStepImp.java:96]: Can you name your parameter what it is?

File: src/test/java/ANALYSIS_TOOLS/JPSurv/stepsImplementations/JPSurvHomePagePlaywrightStepImp.java [hunk 2]
```diff
@@ -89,4 +89,16 @@ public static void selectBothNhLAndCHL() {
...
+     */
+    public static void checkRelaxProportionalityAndCutPoint(String string) {
```
Comment (matakevin) [inline src/test/java/ANALYSIS_TOOLS/JPSurv/stepsImplementations/JPSurvHomePagePlaywrightStepImp.java:98]: Same here.  You string param should be labeled what its for

**Review by matakevin — CHANGES_REQUESTED** [review]
Please review the requested changes.

**Pull request merged**